pkg/apis/velero/v1/constants.go

```diff
@@ -36,4 +36,8 @@ const (
 	// NamespaceScopedDir is the name of the directory containing namespace-scoped
 	// resource within a Velero backup.
 	NamespaceScopedDir = "namespaces"
+
+	// PreferredVersionDir is the sufix name of the directory containing the preferred version of the API group
```

Comment:
Thanks for this; it's helpful if we ever need to do string matching to trim it off.